examples/15-serving-source-pipeline.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: simple-pipeline
+spec:
+  templates:
+    vertex:
+      metadata:
+        annotations:
+          numaflow.numaproj.io/callback: "true"
```

Comment:
We will have to differentiate between a callback store and a serving output store.